yaksh/views.py

```diff
@@ -859,7 +859,9 @@ def get_questions(paper):
         q_unanswered = paper.get_unanswered_questions()
         q_unanswered.sort()
         to_attempt = q_unanswered
-    for index, value in enumerate(all_questions, 1):
+    
+    question_objs = Question.objects.filter(id__in=all_questions)
```

Comment:
You can remove _objs from the above variable name.
Can give other name, also note that you already have questions variable below.